src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -68,6 +70,8 @@ internal override sealed JsonParameterInfo CreateJsonParameterInfo()
         /// </summary>
         internal bool CanBeNull { get; }
 
+        internal readonly EqualityComparer<T> _defaultComparer = EqualityComparer<T>.Default;
```

Comment:
is this used?

Reply:
Yes in JsonPropertyInfo `WriteValue` and `Write`. I believe you refactored this earlier in master; not sure if this branch is up-to-date with master to have that change yet.